fetch_data/sparql_data/create_connectivity_data_observation.py

```diff
@@ -43,15 +42,15 @@ async def create_connectivity_observation(wikibase_id: int) -> bool:
 
 
 async def compile_connectivity_observation(
-    sparql_endpoint_url: str,
+    wikibase: WikibaseModel,
```

Comment:
I did not expect this change based on the PR title.

Reply:
I needed to pass the full Wikibase into this method, so that I could specify the Wikibase in the logs